### PR TITLE
Remove the JSS usage in the codebase.#7363

### PR DESCRIPTION
**Pull request by yogeshmahajan-1903**
Known issue - 
1.About dialogue settings area has blank space after maximise.
2.Terminate button in Process tabs has incorrect color.

### CODE REVIEW

File: web/pgadmin/static/js/SchemaView/DataGridView.jsx [hunk 1]
```diff
@@ -280,16 +266,16 @@ function DataTableRow({index, row, totalRows, isResizing, isHovered, schema, sch
 }
 
 export function DataGridHeader({label, canAdd, onAddClick, canSearch, onSearchTextChange}) {
-  const classes = useStyles();
+
   const [searchText, setSearchText] = useState('');
 
   return (
-    <Box className={classes.gridHeader}>
+    <StyledBox className='DataGridView-gridHeader'>
```
Comment (adityatoshniwal) [inline web/pgadmin/static/js/SchemaView/DataGridView.jsx:273]: why is StyledBox needed here?

File: web/pgadmin/static/js/SchemaView/FormView.jsx [hunk 2]
```diff
@@ -26,44 +25,53 @@ import { useOnScreen } from '../custom_hooks';
...
+      flexDirection: 'column',
+      '& .FormView-sqlTabInput': {
+        border: 0,
+      },
+    }
   },
-  nonTabPanel: {
-    padding: 0,
-    background: 'inherit',
+  '&& .FormView-nonTabPanel': {
```
Comment (adityatoshniwal) [inline web/pgadmin/static/js/SchemaView/FormView.jsx:59]: && should be &

File: web/pgadmin/static/js/SchemaView/index.jsx [hunk 3]
```diff
@@ -784,39 +798,39 @@ function SchemaDialogView({
 
   /* I am Groot */
   return (
-    <StateUtilsContext.Provider value={stateUtils}>
-      <DepListenerContext.Provider value={depListenerObj.current}>
-        <Box className={classes.root}>
-          <Box className={classes.form}>
+    <StyledBox className='Dialog-root'>
```
Comment (adityatoshniwal) [inline web/pgadmin/static/js/SchemaView/index.jsx:801]: no need to have class on root if it is not conditional

File: web/pgadmin/static/js/SecurityPages/BasePage.jsx [hunk 4]
```diff
@@ -79,18 +83,18 @@ export default function BasePage({pageImage, title,  children, messages}) {
     });
   }, [messages]);
   return (
-    <Box className={classes.root}>
+    <StyledBox className='BasePage-root'>
```
Comment (adityatoshniwal) [inline web/pgadmin/static/js/SecurityPages/BasePage.jsx:86]: no need to have class on root if it is not conditional

File: web/pgadmin/static/js/Theme/overrides/reactmenu.override.js [hunk 5]
```diff
@@ -0,0 +1,38 @@
+/////////////////////////////////////////////////////////////
```
Comment (adityatoshniwal) [inline web/pgadmin/static/js/Theme/overrides/reactmenu.override.js:1]: file name should `szhmenu.override.js`

File: web/pgadmin/static/js/components/CheckBoxTree.jsx [hunk 6]
```diff
@@ -46,7 +50,7 @@ export default function CheckBoxTree({treeData, ...props}) {
   }, [checked]);
 
   return (
-    <div className={classes.treeRoot}>
+    <StyledDiv className='CheckBoxTree-treeRoot'>
```
Comment (adityatoshniwal) [inline web/pgadmin/static/js/components/CheckBoxTree.jsx:53]: no need to add class to root if it is not conditional

File: web/pgadmin/static/js/components/EmptyPanelMessage.jsx [hunk 7]
```diff
@@ -1,28 +1,28 @@
...
-  },
+  }
 }));
 
 export default function EmptyPanelMessage({text, style}) {
-  const classes = useStyles();
+
   return (
-    <Box className={classes.root} style={style}>
+    <StyledBox className='EmptyPanelMessage-root' style={style}>
```
Comment (adityatoshniwal) [inline web/pgadmin/static/js/components/EmptyPanelMessage.jsx:22]: no need to add class to root if it is not conditional

File: web/pgadmin/static/js/components/FieldSet.jsx [hunk 8]
```diff
@@ -1,30 +1,33 @@
...
-  }
 }));
 
 export default function FieldSet({title='', className, children}) {
-  const classes = useStyles();
+
   return (
-    <fieldset className={clsx(classes.fieldset, className)}>
-      <legend className={classes.legend}>{title}</legend>
+    <StyledFieldset className={'FieldSet-root ' +  className}>
```
Comment (adityatoshniwal) [inline web/pgadmin/static/js/components/FieldSet.jsx:27]: no need to add class to root if it is not conditional

File: web/pgadmin/static/js/components/FormComponents.jsx [hunk 9]
```diff
@@ -1115,15 +1110,17 @@ PlainString.propTypes = {
...
   /* If raw, then remove the styles and icon */
   return (
-    <Box className={className}>
-      <Paper elevation={0} className={controlProps?.raw ? '' : classes.noteRoot}>
-        {!controlProps?.raw && <Box paddingRight="0.25rem"><DescriptionIcon fontSize="small" /></Box>}
-        <Box>{HTMLReactParse(text || '')}</Box>
-      </Paper>
-    </Box>
+    <Root>
+      <Box className={className}>
```
Comment (adityatoshniwal) [inline web/pgadmin/static/js/components/FormComponents.jsx:1117]: Can't we directly style the Box here?

File: web/pgadmin/static/js/components/FormComponents.jsx [hunk 10]
```diff
@@ -1132,76 +1129,79 @@ FormNote.propTypes = {
...
 /* The form footer used mostly for showing error */
 export function FormFooterMessage({style, ...props}) {
-  const classes = useStylesFormFooter();
-
   if (!props.message) {
     return <></>;
   }
   return (
-    <Box className={classes.root} style={style}>
+    <StyledBox className='FormFooter-root' style={style}>
```
Comment (adityatoshniwal) [inline web/pgadmin/static/js/components/FormComponents.jsx:1202]: no need to add class to root if it is not conditional

**Review by adityatoshniwal — CHANGES_REQUESTED** [review]
Didn't review all, as similar changes required everywhere.

Remove `@mui/styles` package as well.

File: web/pgadmin/static/js/Dialogs/ChangePasswordContent.jsx [hunk 11]
```diff
@@ -73,17 +76,11 @@ class ChangePasswordSchema extends BaseUISchema {
...
-  },
-}));
-
 export default function ChangePasswordContent({getInitData=() => { /*This is intentional (SonarQube)*/ },
   onSave, onClose, hasCsrfToken=false, showUser=true}) {
-  const classes = useStyles();
 
-  return<SchemaView
+
+  return<StyledSchemaView
```
[the remark this answers is not in the document]
Comment (yogeshmahajan-1903) [reply web/pgadmin/static/js/Dialogs/ChangePasswordContent.jsx:83]: Done.

File: web/pgadmin/static/js/SchemaView/FormView.jsx (hunk 2 above)
Comment (yogeshmahajan-1903) [reply web/pgadmin/static/js/SchemaView/FormView.jsx:59]: This is to increase specificity.(To fix background issue for non-tab panel content)

File: web/pgadmin/static/js/Theme/overrides/reactmenu.override.js (hunk 5 above)
Comment (yogeshmahajan-1903) [reply web/pgadmin/static/js/Theme/overrides/reactmenu.override.js:1]: Done.

File: web/pgadmin/about/static/js/AboutComponent.jsx [hunk 12]
```diff
@@ -8,35 +8,34 @@
...
 import getApiInstance from '../../../static/js/api_instance';
 import { copyToClipboard } from '../../../static/js/clipboard';
 import { useDelayedCaller } from '../../../static/js/custom_hooks';
 import { usePgAdmin } from '../../../static/js/BrowserComponent';
 
-
-const useStyles = makeStyles((theme)=>({
-  container: {
+const StyledBox = styled(Box)(({theme}) => ({
+  '&.AboutComponent-container': {
```
[the remark this answers is not in the document]
Comment (yogeshmahajan-1903) [reply web/pgadmin/about/static/js/AboutComponent.jsx:23]: Done

File: web/pgadmin/dashboard/static/js/Dashboard.jsx [hunk 13]
```diff
@@ -834,11 +791,11 @@ function Dashboard({
   };
 
   return (
-    <>
+    (<Root className='Dashboard-root'>
```
[the remark this answers is not in the document]
Comment (yogeshmahajan-1903) [reply web/pgadmin/dashboard/static/js/Dashboard.jsx:794]: Done

File: web/pgadmin/dashboard/static/js/SystemStats/CPU.jsx [hunk 14]
```diff
@@ -246,40 +251,42 @@ CPU.propTypes = {
...
-            caveTable={false}
-            tableNoBorder={false}
-          ></PgTable>
-        </SectionContainer>
-      </Box>
-    </Box>
+        <Box flexGrow={1} minHeight={0}>
+          <SectionContainer title={gettext('Process CPU usage')}>
+            <PgTable
+              className='CPU-autoResizer'
```
[the remark this answers is not in the document]
Comment (yogeshmahajan-1903) [reply web/pgadmin/dashboard/static/js/SystemStats/CPU.jsx:278]: Done

File: web/pgadmin/dashboard/static/js/SystemStats/Memory.jsx [hunk 15]
```diff
@@ -22,38 +22,43 @@ import { getStatsUrl, transformData, statsReducer, X_AXIS_LENGTH } from './utili
...
+      padding: '6px',
+      width: '100%',
+      '& .Memory-containerHeader': {
+        fontSize: '15px',
+        fontWeight: 'bold',
+        display: 'flex',
+        alignItems: 'center',
+        height: '100%',
+      },
+      '& .Memory-autoResizer': {
```
[the remark this answers is not in the document]
Comment (yogeshmahajan-1903) [reply web/pgadmin/dashboard/static/js/SystemStats/Memory.jsx:50]: Done.

File: web/pgadmin/dashboard/static/js/SystemStats/Storage.jsx [hunk 16]
```diff
@@ -21,52 +21,34 @@ import axios from 'axios';
...
-    fontWeight: 'bold',
-    padding: '4px 8px',
-  },
-  tableContainer: {
+
+const Root = styled('div')((
+  {
+    theme
+  }
+) => ({
```
[the remark this answers is not in the document]
Comment (yogeshmahajan-1903) [reply web/pgadmin/dashboard/static/js/SystemStats/Storage.jsx:32]: Done.

File: web/pgadmin/dashboard/static/js/WelcomeDashboard.jsx [hunk 17]
```diff
@@ -131,17 +150,17 @@ function AddNewServer(pgBrowser) {
...
   return (
-    <div className={classes.emptyPanel}>
-      <div className={classes.dashboardContainer}>
-        <div className={classes.row}>
-          <div className={classes.cardColumn}>
-            <div className={classes.card}>
-              <div className={classes.cardHeader}>{gettext('Welcome')}</div>
-              <div className={classes.cardBody}>
-                <div className={classes.welcomeLogo}>
+    <Root className='WelcomeDashboard-emptyPanel'>
```
[the remark this answers is not in the document]
Comment (yogeshmahajan-1903) [reply web/pgadmin/dashboard/static/js/WelcomeDashboard.jsx:156]: Done.

File: web/pgadmin/dashboard/static/js/components/RefreshButtons.jsx [hunk 18]
```diff
@@ -8,36 +8,40 @@
...
 
-  return(
-    <PgIconButton
+
+  return (
+    <StyledPgIconButton
       size="xs"
       noBorder
-      className={classes.refreshButton}
+      className='RefreshButtons'
```
[the remark this answers is not in the document]
Comment (yogeshmahajan-1903) [reply web/pgadmin/dashboard/static/js/components/RefreshButtons.jsx:38]: Done.

File: web/pgadmin/dashboard/static/js/components/SectionContainer.jsx [hunk 19]
```diff
@@ -22,35 +25,35 @@ const useStyles = makeStyles((theme) => ({
...
 
 export default function SectionContainer({title, titleExtras, children, style}) {
-  const classes = useStyles();
+
 
   return (
-    <Box className={classes.root} style={style}>
-      <Box className={classes.cardHeader} title={title}>
-        <div className={classes.cardTitle}>{title}</div>
+    <StyledBox className='SectionContainer-root' style={style}>
```
[the remark this answers is not in the document]
Comment (yogeshmahajan-1903) [reply web/pgadmin/dashboard/static/js/components/SectionContainer.jsx:46]: Done.

File: web/pgadmin/misc/bgprocess/static/js/BgProcessNotify.jsx [hunk 20]
```diff
@@ -1,61 +1,64 @@
...
   },
 }));
 
 function ProcessNotifyMessage({title, desc, onClose, onViewProcess, success=true, dataTestSuffix=''}) {
-  const classes = useStyles();
+
   return (
-    <Box className={clsx(classes.container, (success ? classes.containerSuccess : classes.containerError))} data-test={'process-popup-' + dataTestSuffix}>
-      <Box display="flex" justifyContent="space-between" className={classes.containerHeader}>
+    <StyledBox className={ 'BgProcessNotify-container ' + (success ? 'BgProcessNotify-containerSuccess' : 'BgProcessNotify-containerError')} data-test={'process-popup-' + dataTestSuffix}>
```
[the remark this answers is not in the document]
Comment (yogeshmahajan-1903) [reply web/pgadmin/misc/bgprocess/static/js/BgProcessNotify.jsx:56]: Done.

File: web/pgadmin/misc/bgprocess/static/js/ProcessDetails.jsx [hunk 21]
```diff
@@ -149,15 +158,15 @@ export default function ProcessDetails({data}) {
 
   const errRe = new RegExp(': (' + gettext('error') + '|' + gettext('fatal') + '):', 'i');
   return (
-    <Box display="flex" flexDirection="column" className={classes.container} data-test="process-details">
+    <StyledBox display="flex" flexDirection="column" className='ProcessDetails-container' data-test="process-details">
```
[the remark this answers is not in the document]
Comment (yogeshmahajan-1903) [reply web/pgadmin/misc/bgprocess/static/js/ProcessDetails.jsx:161]: Done.

File: web/pgadmin/misc/bgprocess/static/js/Processes.jsx [hunk 22]
```diff
@@ -126,46 +99,50 @@ export default function Processes() {
...
+        </Root>
       );
     };
     LogsCell.displayName = 'LogsCell';
     LogsCell.propTypes = cellPropTypes;
 
     const StatusCell = ({row})=>{
       const [text, bgcolor] = ProcessStateTextAndColor[row.original.process_state];
-      return <Box className={classes[bgcolor]}>{text}</Box>;
+      return <Root><Box className={'Processes-'+bgcolor}>{text}</Box></Root>;
```
[the remark this answers is not in the document]
Comment (yogeshmahajan-1903) [reply web/pgadmin/misc/bgprocess/static/js/Processes.jsx:145]: Removed.

File: web/pgadmin/misc/file_manager/static/js/components/FileManager.jsx [hunk 23]
```diff
@@ -694,23 +711,23 @@ export default function FileManager({params, closeModal, onOK, onCancel, sharedS
 
   return (
     <ErrorBoundary>
-      <Box display="flex" flexDirection="column" height="100%" className={modalClasses.container}>
+      <StyledBox display="flex" flexDirection="column" height="100%" className='FileManager-container'>
```
[the remark this answers is not in the document]
Comment (yogeshmahajan-1903) [reply web/pgadmin/misc/file_manager/static/js/components/FileManager.jsx:714]: Done.

File: web/pgadmin/misc/file_manager/static/js/components/Uploader.jsx [hunk 24]
```diff
@@ -173,18 +171,18 @@ export default function Uploader({fmUtilsObj, onClose}) {
   }, [files.length]);
 
   return (
-    <Box className={classes.root}>
+    <StyledBox className='Uploader-root'>
```
[the remark this answers is not in the document]
Comment (yogeshmahajan-1903) [reply web/pgadmin/misc/file_manager/static/js/components/Uploader.jsx:174]: Done.

File: web/pgadmin/misc/properties/CollectionNodeProperties.jsx [hunk 25]
```diff
@@ -309,13 +295,13 @@ export default function CollectionNodeProperties({
   return (
     <Theme className='obj_properties'>
       <Loader message={loaderText}/>
-      <Box className={classes.propertiesPanel}>
+      <StyledBox className='CollectionNodeProperties-propertiesPanel'>
```
[the remark this answers is not in the document]
Comment (yogeshmahajan-1903) [reply web/pgadmin/misc/properties/CollectionNodeProperties.jsx:298]: Done.

File: web/pgadmin/misc/statistics/static/js/Statistics.jsx [hunk 26]
```diff
@@ -88,10 +72,10 @@ function getColumn(data, singleLineStatistics, prettifyFields=[]) {
   }
   columns.forEach((c)=>{
     // Prettify the cell view
-    if(prettifyFields.includes(c.header)) {
-      c.cell = ({value})=><>{toPrettySize(value)}</>;
-      c.cell.displayName = 'Cell';
-      c.cell.propTypes = {
+    if(prettifyFields.includes(c.Header)) {
```
[the remark this answers is not in the document]
Comment (yogeshmahajan-1903) [reply web/pgadmin/misc/statistics/static/js/Statistics.jsx:75]: Done.

File: web/pgadmin/static/js/components/FieldSet.jsx (hunk 8 above)
Comment (yogeshmahajan-1903) [reply web/pgadmin/static/js/components/FieldSet.jsx:27]: Done.

File: web/pgadmin/static/js/Explain/Analysis.jsx [hunk 27]
```diff
@@ -165,46 +160,50 @@ export default function Analysis({explainTable}) {
...
-          show_timings: explainTable.show_timings,
-          show_rowsx: explainTable.show_rowsx,
-          show_rows: explainTable.show_rows,
-          show_plan_rows: explainTable.show_plan_rows,
-        }} activeExId={activeExId} setActiveExId={setActiveExId} collapsedExId={collapsedExId} toggleCollapseExId={toggleCollapseExId} />;
-      })}
-    </tbody>
-  </table>;
+  return (
+    <Root>
```
[the remark this answers is not in the document]
Comment (yogeshmahajan-1903) [reply web/pgadmin/static/js/Explain/Analysis.jsx:164]: Done.

File: web/pgadmin/static/js/PgTreeView/index.jsx [hunk 28]
```diff
@@ -173,17 +175,19 @@ function Node({ node, style, tree, hasCheckbox, onNodeSelectionChange}) {
...
-      {
-        hasCheckbox ? <Checkbox style={{ padding: 0 }} color="primary" className={clsx(!node.isInternal ? classes.leafNode: null)}
-          checked={isSelected}
-          checkedIcon={isIndeterminate  ? <IndeterminateCheckBoxIcon style={{height: '1.4rem'}} />: <CheckBoxIcon style={{height: '1.4rem'}} />}
-          onChange={onCheckboxSelection}/> :
-          <span className={clsx(node.data.icon)}></span>
-      }
-      <div className={clsx(node.data.icon, classes.node)}>{node.data.name}</div>
-    </div>
+    <Root>
```
[the remark this answers is not in the document]
Comment (yogeshmahajan-1903) [reply web/pgadmin/static/js/PgTreeView/index.jsx:178]: Done.

File: web/pgadmin/static/js/SecurityPages/BasePage.jsx (hunk 4 above)
Comment (yogeshmahajan-1903) [reply web/pgadmin/static/js/SecurityPages/BasePage.jsx:86]: Done.

File: web/pgadmin/static/js/QuickSearch/index.jsx [hunk 29]
```diff
@@ -44,32 +58,21 @@ const useStyles = makeStyles((theme)=>({
...
 }));
 
 function SearchLoader({loading=false}) {
-  const classes = useStyles();
+
   if(loading) {
     return (
-      <div className={classes.loaderRoot}>
-        <CircularProgress className={classes.loader} />
+      <StyledDiv className='QuickSearch-loaderRoot'>
```
[the remark this answers is not in the document]
Comment (yogeshmahajan-1903) [reply web/pgadmin/static/js/QuickSearch/index.jsx:72]: Removed styleddiv from here.

File: web/pgadmin/static/js/components/EmptyPanelMessage.jsx (hunk 7 above)
Comment (yogeshmahajan-1903) [reply web/pgadmin/static/js/components/EmptyPanelMessage.jsx:22]: Done.

File: web/pgadmin/static/js/components/FormComponents.jsx (hunk 10 above)
Comment (yogeshmahajan-1903) [reply web/pgadmin/static/js/components/FormComponents.jsx:1202]: Done.

File: web/pgadmin/static/js/SchemaView/index.jsx (hunk 3 above)
Comment (yogeshmahajan-1903) [reply web/pgadmin/static/js/SchemaView/index.jsx:801]: Done

File: web/pgadmin/static/js/Dialogs/ConnectServerContent.jsx [hunk 30]
```diff
@@ -58,74 +58,76 @@ export default function ConnectServerContent({closeModal, data, onOK, setHeight}
...
+            <Box marginTop='12px'>
+              <InputCheckbox controlProps={{label: gettext('Save Password')}} value={formData['save_password']}
+                onChange={(e)=>onTextChange(e.target.checked, 'save_password')} disabled={!data.allow_save_password} />
+            </Box>
+          </>}
+          <FormFooterMessage type={MESSAGE_TYPE.ERROR} message={_.escape(data.errmsg)} closable={false} style={{
+            position: 'unset', padding: '12px 0px 0px'
+          }}/>
+        </Box>
+        <Box className='ModalContent-footer'>
```
[the remark this answers is not in the document]
Comment (yogeshmahajan-1903) [reply web/pgadmin/static/js/Dialogs/ConnectServerContent.jsx:109]: Done.

File: web/pgadmin/static/js/Dialogs/MasterPasswordContent.jsx [hunk 31]
```diff
@@ -59,81 +56,83 @@ export default function MasterPasswordContent({ closeModal, onResetPassowrd, onO
...
-            <br />
-            <span style={{ fontWeight: 'bold' }}>
-              {isPWDPresent ? gettext('This is required to unlock saved passwords and reconnect to the database server(s).') : gettext('This will be used to secure and later unlock saved passwords and other credentials.')}
-            </span>
-          </Box>
-          <Box marginTop='12px'>
-            <InputText inputRef={firstEleRef} type="password" value={formData['password']} maxLength={null}
-              onChange={(e) => onTextChange(e, 'password')} onKeyDown={(e) => onKeyDown(e)} controlProps={{autoComplete: 'new-password'}}/>
+    <ModalContent>
+      <Box display="flex" flexDirection="column" className='ModalContent-container' ref={containerRef}>
```
[the remark this answers is not in the document]
Comment (yogeshmahajan-1903) [reply web/pgadmin/static/js/Dialogs/MasterPasswordContent.jsx:59]: Done.

File: web/pgadmin/static/js/Dialogs/NamedRestoreContent.jsx [hunk 32]
```diff
@@ -55,30 +54,32 @@ export default function NamedRestoreContent({closeModal, onOK, setHeight}) {
...
 
   return (
-    <Box display="flex" flexDirection="column" className={classes.container} ref={containerRef}>
-      <Box flexGrow="1" p={2}>
-        <Box>
-          <span style={{fontWeight: 'bold'}}>
-            {gettext('Enter the name of the restore point to add')}
-          </span>
+    <ModalContent>
+      <Box display="flex" flexDirection="column" className='ModalContent-container' ref={containerRef}>
```
[the remark this answers is not in the document]
Comment (yogeshmahajan-1903) [reply web/pgadmin/static/js/Dialogs/NamedRestoreContent.jsx:58]: Done.

File: web/pgadmin/static/js/Dialogs/ConnectServerContent.jsx [hunk 33]
```diff
@@ -58,74 +58,76 @@ export default function ConnectServerContent({closeModal, data, onOK, setHeight}
...
+          <DefaultButton data-test="close" startIcon={<CloseIcon />} onClick={()=>{
+            closeModal();
+          }} >{gettext('Cancel')}</DefaultButton>
+          <PrimaryButton ref={okBtnRef} data-test="save" className='ModalContent-Btnmargin' startIcon={<CheckRoundedIcon />} onClick={()=>{
```
[the remark this answers is not in the document]
Comment (yogeshmahajan-1903) [reply web/pgadmin/static/js/Dialogs/ConnectServerContent.jsx:113]: Done.

**Comment by yogeshmahajan-1903** [comment]
> Didn't review all, as similar changes required everywhere.
> 
> Remove `@mui/styles` package as well.

Done.

File: web/pgadmin/static/js/components/FormComponents.jsx (hunk 9 above)
Comment (yogeshmahajan-1903) [reply web/pgadmin/static/js/components/FormComponents.jsx:1118]: Need to add another styling element then.

File: web/pgadmin/misc/properties/CollectionNodeProperties.jsx [hunk 34]
```diff
@@ -309,13 +295,13 @@ export default function CollectionNodeProperties({
   return (
     <Theme className='obj_properties'>
```
[the remark this answers is not in the document]
Comment (yogeshmahajan-1903) [reply web/pgadmin/misc/properties/CollectionNodeProperties.jsx:276]: Done

**Review by adityatoshniwal — APPROVED** [review]

**Pull request merged**